package dev.vality.disputes.tg.bot.common.exception;

public class DisputesApiException extends RuntimeException {

    public DisputesApiException(String message, Throwable cause) {
        super(message, cause);
    }
}
